src/main/java/org/jboss/remoting3/remote/RemoteConnectionHandler.java

```diff
@@ -418,7 +416,6 @@ public String getRemoteEndpointName() {
 
     protected void closeAction() throws IOException {
         sendCloseRequest();
-        remoteConnection.shutdownWrites();
```

Comment:
If we don't shut down writes here (and up above in close()) then can you show that we won't leak connections?